src/Components/test/E2ETest/ServerExecutionTests/ServerSideAppTest.cs

```diff
@@ -23,12 +24,16 @@ public class ServerSideAppTest : ServerTestBase<AspNetSiteServerFixture>
         {
             _serverFixture.Environment = AspNetEnvironment.Development;
             _serverFixture.BuildWebHostMethod = ComponentsApp.Server.Program.BuildWebHost;
+        }
+
+        public override async Task InitializeAsync()
+        {
+            await base.InitializeAsync();
```

Comment:
Does it make more sense to use the `InitializeAsync`/`InitializeCoreAsync` pattern? this is error prone

Reply:
Yeah, will change it.

Reply:
Done